meadow-client/src/Simulation.purs

```diff
@@ -448,7 +448,13 @@ printTransError AmbiguousId = [ul_ [li_ [text "At least one of the action identi
 printTransError InternalError = [ul_ [li_ [text "The internal state of the contract is inconsistent. This should not happen. Please, open an issue and let us know how you got to this error."]]]
 
 contractParsingError :: forall p. Boolean -> Array (HTML p Query)
-contractParsingError true = []
+contractParsingError true = [div [classes [ ClassName "invalid-transaction"
+                                           , ClassName "input-composer"
+                                           ]
+                                  ]
+                                  [ h2 [] [text ""]
```

Comment:
You need some whitespace that's guaranteed to be the same height as an `h2` somewhere else??

Reply:
this is a temporary fix, Richie is going to take a look at a proper solution tomorrow but without this it was effectively impossible to use as the editor kept loosing focus. Also, weirdly the height changes when the h2 is empty but it stops the editor losing focus, no idea why.